### DISCUSSION TITLE
Daily Standup - Thursday, April 20

### DISCUSSION THREAD

**Discussion by antonieto** [antonieto/ibm-dashboard#140] [General]
Share your daily standup update with the team here. Respond to this
discussion before the standup starts in the following format:

```
**What did you achieve in the last 24 hours?**:
 - 

**What are your priorities for the next 24 hours?**:
 - 

**Blockers**:
 -

**Shoutouts**:
 - @username for x
```

**Comment by DavBot02** [comment]
**What did you achieve in the last 24 hours?**:
 - Finished auth tutorial

**What are your priorities for the next 24 hours?**:
 - Make first contribution

**Blockers**:
 - None

**Shoutouts**:
 - @Int-Angel

**Comment by antonieto** [comment]
**What did you achieve in the last 24 hours?**:
 - Added the following issues with @Int-Angel 
   - #139 
   - #138 
   - #137 
   - #136 

**What are your priorities for the next 24 hours?**:
 - start working on #138 and #137 

**Blockers**:
 - none atm

**Shoutouts**:
 - @DavBot02 because

**Comment by antonieto** [comment]
# @Jhoscar22's update
**What did you achieve in the last 24 hours?**:
 - Check out passport stuff

**What are your priorities for the next 24 hours?**:
 - Finish Signup

**Blockers**:
 - No blockers
 - Maybe define DB stuff

**Shoutouts**:
 - @everyone for doing the docker thing
